quickstarts/minikube/index.md

```diff
@@ -0,0 +1,44 @@
+# Starting Minikube 
+
+This assumes that you have the latest version of the `minikube` binary, which you can get [here](https://kubernetes.io/docs/setup/minikube/#installation).
+
+```shell
+minikube start --memory=8192 --cpus=4 \
+  --kubernetes-version=v1.12.1 \
+  --vm-driver=kvm2 \
```

Comment:
This is not really portable to other operating systems. I would leave this out. Most users have the VM configured to what they want or they don't care.

Reply:
@scholzj how about just saying `minikube start.....` with your configured VM. If need help look DOC for more ?

Reply:
Yeah, that sounds good to me.